include/Util/Keycode.hpp

```diff
@@ -398,10 +397,19 @@ enum class Keycode {
 
     /* Add any other keys here. */
 
-    PTSD_NUM_SCANCODES = SDL_NUM_SCANCODES /**< not a key, just marks the number of scancodes
+    PTSD_NUM_SCANCODES = SDL_NUM_SCANCODES, /**< not a key, just marks the number of scancodes
                                  for array bounds */
+
+
+    // Below are the keycodes handling mouse buttons.
+
+    PTSD_MOUSE_LB = 513,
+    PTSD_MOUSE_RB = 514,
+    PTSD_MOUSE_MB = 515,
```

Comment:
do we need `PTSD_` prefix here since we didn't add it for other keycodes

Reply:
Fixed